### DISCUSSION TITLE
Indent guide highlight not working

### DISCUSSION THREAD

**Discussion by vikasguptadev** [ajaxorg/ace#5619] [Q&A] [answered]
I am using 1.35.4 of ace editor CDN version. I am trying to highlight the indent guides but no success. Below is the code: 

```
editor.setOptions({
				foldStyle: 'markbeginend',  //This works
				enableBasicAutocompletion: true,  //This works
				enableLiveAutocompletion: true,  //This works
				showPrintMargin: false,  //This works
				displayIndentGuides: true,  //This works
				highlightIndentGuides: true  //This does not works
			});
```			

I have also tried `editor.setHighlightIndentGuides(true);` but no success.

I inspected the HTML and could not see the class `ace_indent-guide-active` being set.

**Comment by vikasguptadev** [comment, marked as answer]
Found that if UseWrapMode = true then highlightIndentGuides will not work. 

Should have been mentioned in the doc

**Comment by nlujjawal** [comment]
vikasguptadev we have released a new version of ace v1.38.0 which should be fixing this issue.
